.github/actions/debug-info/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Install RcppDeepState with its dependendcies
+Rscript -e "install.packages(c('Rcpp', 'RcppArmadillo', 'RInside', 'data.table', 'qs', 'nc', 'xml2'), repos='http://cran.us.r-project.org')"
+Rscript -e "install.packages('./', repos = NULL, type='source', verbose=TRUE)"
+
+# Compile the TestSAN package
+Rscript -e "RcppDeepState::deepstate_harness_compile_run('./inst/testpkgs/testSAN')"
+
+# check if the compiled binaries contains debug symbols
+debug_info=$(file ./inst/testpkgs/testSAN/src/read_out_of_bound.o | grep debug_info)
+if [ -z "$debug_info" ] ; then
+    echo "ERROR: The binaries are missing debug symbols."
```

Comment:
@FabrizioSandri can this logic be moved to to a tests/testthat/test-*.R file?

Reply:
Yes, I'm sorry, I already moved it to the testthat file, but I forgot to delete the old Action files located in the `.github/actions/debug-info` folder. I completely replaced that action with a new workflow that uses [r-lib/actions](https://github.com/r-lib/actions) and runs `R CMD check` to run testthat. I developed a new test in the file `tests/testthat/test-debug-symbols.R` that checks if the result table is empty when using a seed that is known to detect certain errors(seed=1000). The result table of the analysis will be empty if debug symbols are not included in the compilation stages.
 
I'll remove this old files in the next commit.